### PR TITLE
Convert FileUtilsInstrumentationTest.java into kotlin

### PR DESCRIPTION
**Pull request by MohitMaliFtechiz**
Fixes: #3054 
Parent Issue: #1826 

Converted FileUtilsInstrumentationTest.java into kotlin

### CODE REVIEW

File: app/src/androidTest/java/org/kiwix/kiwixmobile/utils/files/FileUtilsInstrumentationTest.kt [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+import org.kiwix.kiwixmobile.core.entity.LibraryNetworkEntity
+import java.io.File
+import java.io.IOException
+import java.util.Random
+
+class FileUtilsInstrumentationTest {
+  private var context: Context? = null
+  private var testDir: File? = null
+  @Before fun executeBefore() {
+    context = InstrumentationRegistry.getTargetContext()
```
Comment (gouri-panda) [inline app/src/androidTest/java/org/kiwix/kiwixmobile/utils/files/FileUtilsInstrumentationTest.kt:37]: This method is deprecated. Can we use this `ApplicationProvider.getApplicationContext()` instead?

**Review by gouri-panda — COMMENTED** [review]
@MohitMaliDeveloper Just some small changes then we are good to go :)

File: app/src/androidTest/java/org/kiwix/kiwixmobile/utils/files/FileUtilsInstrumentationTest.kt (hunk 1 above)
Comment (MohitMaliFtechiz) [reply app/src/androidTest/java/org/kiwix/kiwixmobile/utils/files/FileUtilsInstrumentationTest.kt:37]: hi @gouri-panda ,

we can use ```androidx.test.platform.app.InstrumentationRegistry.getInstrumentation().targetContext``` insted of ```androidx.test.InstrumentationRegistry.getTargetContext()```

```ApplicationProvider.getApplicationContext()```  returns application context ```androidx.test.platform.app.InstrumentationRegistry.getInstrumentation().getTargetContext().getApplicationContext()```

[the remark this answers is not in the document]
Comment (MohitMaliFtechiz) [reply app/src/androidTest/java/org/kiwix/kiwixmobile/utils/files/FileUtilsInstrumentationTest.kt]: done

Comment (gouri-panda) [reply app/src/androidTest/java/org/kiwix/kiwixmobile/utils/files/FileUtilsInstrumentationTest.kt]: @MohitMaliDeveloper Sure use this package's API `androidx.test.platform.app`.

Comment (MohitMaliFtechiz) [reply app/src/androidTest/java/org/kiwix/kiwixmobile/utils/files/FileUtilsInstrumentationTest.kt]: hi @gouri-panda  done

**Review by gouri-panda — APPROVED** [review]

**Pull request merged**